qa/common/gen_qa_sequence_models.py

```diff
@@ -185,8 +185,15 @@ def create_tf_modelfile(create_savedmodel, models_dir, model_version, max_batch,
         acc = tf.get_variable("ACC", [
             1,
         ], dtype=tf_dtype)
-        tmp = tf.where(tf.equal(start0, 1), input0, tf.add(acc, input0))
-        newacc = tf.where(tf.equal(ready0, 1), tmp, acc)
+
+        if tf_dtype == tf.bool:
+            tmp = tf.where(tf.equal(start0, True), input0,
+                           tf.math.logical_and(acc, input0))
+            newacc = tf.where(tf.equal(ready0, True), tmp, acc)
```

Comment:
The bool control models you generate here are different from the models with other control types, that is why you found that you still need to modify the test script at the end.

What you can do is to make the bool control models behave the same as the other models (same I/O types and operators), except for the places where the "control tensors" are used (comparing to `True` instead of `1`). Take this as example, if you have the following, then from the Triton client's perspective, the model should be identical to the model with other control type.
```
if tf_dtype == tf.bool:
            tmp = tf.where(tf.equal(start0, True), input0, tf.add(acc, input0))
            newacc = tf.where(tf.equal(ready0, True), tmp, acc)
```